src/transformers/models/auto/modeling_auto.py

```diff
@@ -29,7 +29,57 @@
 
 logger = logging.get_logger(__name__)
 
-
+_MODEL_FOR_IMAGE_MAPPING_NAMES = OrderedDict(
```

Comment:
Should we remove these from `MODEL_MAPPING_NAMES`?

Reply:
No, I don't think so. `MODEL_MAPPING_NAMES` is used for loading with `AutoModel` and I think this might break quite a few things. 

It's fine to have a model listed in more that one mapping e.g. `"CLIPModel"` is listed under both `MODEL_MAPPING_NAMES` and `MODEL_FOR_ZERO_SHOT_IMAGE_CLASSIFICATION_MAPPING_NAMES`